automated install/basic-install.sh

```diff
@@ -1224,15 +1224,15 @@ version_check_dnsmasq() {
     # Local, named variables
     local dnsmasq_conf="/etc/dnsmasq.conf"
     local dnsmasq_conf_orig="/etc/dnsmasq.conf.orig"
-    local dnsmasq_pihole_id_string="addn-hosts=/etc/pihole/gravity.list"
+    local dnsmasq_pihole_id_string="# Dnsmasq config for Pi-hole's FTLDNS"
```

Comment:
Older versions would not have this new string, so the old string should still be used.